trailblazer/store/api.py

```diff
@@ -255,6 +255,13 @@ def set_analysis_uploaded(self, case_id: str, uploaded_at: dt.datetime) -> None:
         self.commit()
         LOG.info(f"{analysis_obj.family} - uploaded at set to {uploaded_at}")
 
+    def set_analysis_failed(self, case_id: str, status: str):
```

Comment:
Misleading function name since it can be used to set to complete as well

Reply:
Nice comment, I guess it would be useful to have a function to set any status so I guess I change it to set_analysis_status